sdk/storage/storage-blob/package.json

```diff
@@ -85,6 +85,7 @@
     "@azure/core-http": "1.0.0-preview.6",
     "@azure/core-paging": "1.0.0-preview.2",
     "@azure/core-tracing": "1.0.0-preview.5",
+    "@azure/core-lro": "1.0.0-preview.1",
```

Comment:
>"@azure/core-lro": [](start = 4, length = 18)

Does this dependency introduces 3rd dependencies, if so, should update in OSS and ThirdPartyNotice.txt

Reply:
It only depends on packages `storage-blob` already depends on, so there should be no change. That said, I don't see a ThirdPartyNotice.txt file in this package...were you maintaining one already?

Reply:
There is one under the browser folder.

@XiaoningLiu We will be doing a sweep of all dependencies before GA and will handle this then

Reply:
@ramya-rao-a Thanks! Better have a work item for tracking in GA. It's also very promising if we could make it automatically in the future mentioned by Harsha.